src/modules/party/dto/post-party-request.dto.ts

```diff
@@ -0,0 +1,45 @@
+import { ApiProperty } from '@nestjs/swagger';
+import { IsISO8601, IsOptional, Length } from 'class-validator';
+
+export type PostPartyRequest = PostPartyRequestItem[];
+
+export class PostPartyRequestItem {
+  @ApiProperty()
+  @Length(3, 20)
+  alternateIdentifier: string;
+
+  @ApiProperty()
+  @Length(1, 10)
```

Comment:
Where have we got the lengths from? Does Mulesoft validate these lengths or are they from ACBS?

Reply:
They are from ACBS. These were found through testing on Postman.

Reply:
> These were found through testing on Postman.
Ah Okay - you can also check the ACBS documentation :)